build.gradle

```diff
@@ -56,5 +57,6 @@ publishPlugin {
 }
 
 repositories {
+    mavenLocal()
```

Comment:
You need to add JBoss Nexus as well if you can CI to succeed

Reply:
@jeffmaury JBoss Nexus is mirrored to/from central, why add a specific source?

Reply:
I seem not to be authorized to publish:
```
> Could not PUT 'https://repository.jboss.org/nexus/content/groups/public/com/redhat/devtools/intellij/intellij-common/0.0.1-SNAPSHOT/maven-metadata.xml'. Received status code 401
```

Reply:
You need to add it for SNAPSHOT versions and I don't think the whole JBoss Nexus is cloned on Maven Central

Reply:
> You need to add it for SNAPSHOT versions 

it is in my `~/.m2/settings.xml` for the url above via 
```
 <releases>
              <enabled>true</enabled>
              <updatePolicy>never</updatePolicy>
           </releases>
           <snapshots>
             <enabled>true</enabled>
             <updatePolicy>never</updatePolicy>
           </snapshots>
```